webapp/src/routes/ExpenseReport/index.js

```diff
@@ -0,0 +1,307 @@
+import React, { memo } from 'react'
+import { makeStyles } from '@mui/styles'
+import {
+  FormControl,
+  FormControlLabel,
+  Radio,
+  RadioGroup,
+  Switch,
+  Tooltip,
+  Typography
+} from '@mui/material'
+import { useTranslation } from 'react-i18next'
+
+import useExpenseReport from '../../hooks/customHooks/useExpenseReportState'
+import LineAreaChartReport from '../../components/LineAreaChartReport'
+import StackedChartReport from '../../components/StackedChartReport'
+import TreasuryBalance from '../../components/TreasuryBalance'
+import PieChartReport from '../../components/PieChartReport'
+import { formatWithThousandSeparator } from '../../utils'
+import TableReport from '../../components/TableReport'
+import SelectComponent from '../../components/Select'
+
+import styles from './styles'
+
+const useStyles = makeStyles(styles)
+
+const rowsCenter = { flex: 1, align: 'center', headerAlign: 'center' }
+
+const ExpenseReport = () => {
+  const classes = useStyles()
+
+  const { t } = useTranslation('expenseRoute')
+
+  const { t: t2 } = useTranslation('generalForm')
```

Comment:
why do you need to rename the translate function?

do you know you can use the `useTranslation` hook without a parameter and in this way get access to entire language props?

So, you can do that: `const { t } = useTranslation()` and if you need to access the `expenseRoute` properties just need to use the `t` function with the complete language property path, for example:

`t('expenseRoute. textInformation')`

And the same way for `generalForm `: 

`t('generalForm. textRadioButton3')`

In short, using the t function with a complete path we are avoiding calling twice the useTranslate hook

Reply:
I tried with the complete path and dont work, but i searched and find [this ](https://react.i18next.com/latest/usetranslation-hook) like next image:
![imagen](https://user-images.githubusercontent.com/49013556/195217813-a81c07c3-0742-4552-99b4-8ba9fb20d311.png)
With this i t2 deleted and work fine

Reply:
I tried this:
`const { t } = useTranslation()`
`headerName: t('tableHeader2', { ns: 'expenseRoute' })`
Where "ns" is the path, and work fine